README.md

```diff
@@ -10,30 +10,46 @@ This project is a [_fastlane_](https://github.com/fastlane/fastlane) plugin. To
+## Usage
+To get started, first, ask your Dynatrace administrator for a an API token - it can be generated in the global settings in "Integration > Dynatrace API". The token needs the permission 'Mobile symbolication file management' and is used by the plugin to obtain permission to upload the symbol and mapping files to Dynatrace.
 
-### Important
-There is a time gap between the application being uploaded to App Store Connect and the dSYM files to be ready. So **_we have to introduce some "wait" time in the CI to accomodate for this_**. You can do this by setting the `waitForDsymProcessing` to true. Unfortunately, Apple does not specify how long this time is. We recommend 1800 seconds (30 mins) as this is usually enough for the symbols are ready for download. You can increase this timeout if needed (`waitForDsymProcessingTimeout`). 
+Add the action `dynatrace_process_symbols` to your Fastfile, see further below for all configuration options and a default configuration.
 
-> Notice that this timeout is only the **maximum** waiting time. If the symbol files are ready sooner, it will continue processing and will not wait for the whole duration of the timeout.
+Now, when you run fastlane, the Dynatrace plugin will manage the symbol files of your app as configured.
+
+### Dynatrace Managed (1.195 and earlier)
+For cluster versions 1.195 and earlier the Dynatrace application 'Symbolication Client' has to be downloaded manually and explicitly specified (`dtxDssClientPath`). For all cluster versions above 1.195 it is fetched and updated automatically. A matching version can be downloaded manually with this link [https://api.mobileagent.downloads.dynatrace.com/sprint-latest-dss-client/xyz](https://api.mobileagent.downloads.dynatrace.com/sprint-latest-dss-client/xyz) by replacing `xyz` with the 3-digit sprint version of your Dynatrace Managed installation.
+
+
+## A) Bitcode-enabled iOS apps
+This is the right option if your app is distributed via Apple's App Store or TestFlight and Bitcode-enabled. For all other cases look at option B below.
+
+Background: If your app is Bitcode-enabled, then the dSYMs that are generated during the Xcode build are **not** the dSYMs that need to be uploaded to Dynatrace. Apple recompiles the application on their servers, generating new dSYM files in the process. These newly generated dSYM files need to be downloaded from *App Store Connect*, processed and uploaded to Dynatrace.
+
+### Automatically downloading dSYMs
+
+To fully automate the following five step workflow, add the snippets below to the respective files and fill in the placeholders:
+
+1. Upload build to App Store Connect
```

Comment:
again, the plugin doesn't upload the app - only waits for processing to finish

so the plugin should be called after uploading the app to apple (with normal fastlane action)